packages/twenty-front/src/modules/ui/input/components/SelectStrategy.tsx

```diff
@@ -0,0 +1,74 @@
+import { MouseEvent, useMemo, useRef, useState } from 'react';
+
+import { SelectValue } from './Select';
+import { SelectSizeVariant, SelectOption } from './Select';
+import { IconComponent } from 'twenty-ui';
+
+type CallToActionButton = {
+  text: string;
+  onClick: (event: MouseEvent<HTMLDivElement>) => void;
+  Icon?: IconComponent;
+};
+
+
+export type SelectObject<Value extends SelectValue> = {

```

Comment:
we try to have one file = one export

We would need to split this between types and constants